api/fossa/locator.go

```diff
@@ -69,8 +80,8 @@ func ReadImportPath(s ImportPathString) ImportPath {
 	return out
 }
 
-func LocatorOf(id pkg.ID) Locator {
-	return Locator{
+func LocatorOf(id pkg.ID) *Locator {
```

Comment:
Why return a pointer here?

Reply:
Because every usage of it I could find immediately called the `.String` method, so why create a new object and consume more memory when the usage just wants the string output anyways.